packages/actor-rdf-parse-html/test/ActorRdfParseHtml-test.ts

```diff
@@ -129,6 +129,7 @@ describe('ActorRdfParseHtml', () => {
           },
         };
 
+        // tslint:disable-next-line:no-unused-expression
```

Comment:
We probably want a more specific `.tslint` for tests that is more relaxed, so we don't need exceptions.